metadata_service/dao/data_access_committee.py

```diff
@@ -104,9 +107,14 @@ async def update_dac(dac_id: str, data: DataAccessCommittee) -> DataAccessCommit
     """
     db_connect = DBConnect()
     collection = await db_connect.get_collection(COLLECTION_NAME)
-    await collection.update_one(  # type: ignore
-        {"id": dac_id}, {"$set": data.dict(exclude_unset=True)}
-    )
+    data.id = dac_id
+    if data.creation_date:
+        data.creation_date = None
```

Comment:
Should we set the created date to None here? I guess we shouldn't bother about the creation date after it has been created and only update the updated date.

Reply:
Yes, here we are doing 2 things:
- set the `creation_date` to `None` (if and only if its part of the incoming data JSON)
- when the API sends the JSON to mongodb, only those fields that are set will be sent (thanks to `exclude_none=True`)

So there is no risk of `creation_date` being set to `None` in mongodb.

Reply:
Should we allow the incoming data to contain the creation_data field? I am wondering if we should stop it at the source and not here..

Reply:
I think its fine since there are only few fields that needs to be constrained. The alternative is to have Partials defined in the models. And that can be brittle.

Reply:
Lets go ahead with this plan and revisit the partials idea after the demo since we do that for the request object